AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -34,6 +34,7 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 /googleplay_135x40.png$domain=xn----etbbecbrbp5ahkja1ae7v.xn--p1ai|kingisepp-today.ru
 /appstore_135x40.png$domain=xn----etbbecbrbp5ahkja1ae7v.xn--p1ai|kingisepp-today.ru
+m.imgur.com##div[class^="SeeImgurOIA-"]
```

Comment:
![image](https://user-images.githubusercontent.com/8361299/82440800-96e39500-9aa5-11ea-8234-f93f0ccca9a2.png)

Reply:
Scrolling works good despite of that attribute

Reply:
Scrolling works good on ios even with this attribute.
We can add something like this
```
!+ PLATFORM(ios)
m.imgur.com##div[class^="SeeImgurOIA-"]
!+ PLATFORM (android)
m.imgur.com#?#div[class^="SeeImgurOIA-"] { display: none!important; }
m.imgur.com#?#body {overflow-y: hidden!important; }
```

Reply:
> We can add something like this

ok